src/components/MarkdownProvider/components/CodeExample/index.tsx

```diff
@@ -89,7 +88,7 @@ export const CodeExample: React.FC<ICodeExampleProps> = ({ children, code }) =>
         border-radius: ${p => p.theme.borderRadii.md};
       `}
     >
-      <ThemeProvider theme={exampleTheme} focusVisibleRef={focusVisibleRef}>
+      <ThemeProvider theme={exampleTheme}>
```

Comment:
Good call-out. We'll definitely have to revise the website color treatment, especially since we intend on adding dark mode. I anticipate to get Design involved once the code migration is complete.

Reply:
The removal of `focusVisibleRef` required modifications for `:focus-visible` CSS treatments. I'm addressing this under a separate PR.